### ISSUE TITLE
Running PowerJoular in WSL

### ISSUE THREAD

**Issue by somak-chatterjee** [joular/powerjoular#68]
Hello! I wanted to ask if it is possible to run PowerJoular on WSL 2?

**Comment by somak-chatterjee** [comment]
Hi! Thanks for the response! I did try out Powerjoular on WSL and got some readings for GPU power usage. I wasn't able to get any readings for the CPU. Perhaps, the fact that Powerjoular can't access the RAPL interface, is the reason why? 

Anyway, I posted the screenshot of my output. Maybe this could provide some useful info. I'll also take a look at the workaround using the VM. Thanks again!

![powerjoular_output](https://github.com/user-attachments/assets/ce2093fe-b4fd-4851-a031-e9f3b41f87a4)
